### DISCUSSION TITLE
Skripte werden nicht (nach)geladen

### DISCUSSION THREAD

**Discussion by ynamite** [FriendsOfREDAXO/consent_manager#222] [Q&A]
Funzt soweit eigentlich alles, nur werden die Skripte nach Bestätigung der Auswahlbox nicht geladen.

Habe die Demo-Daten importiert, Domain eingerichtet und zugewiesen, Platzhalter im Template gesetzt.
Wenn ich beim GA Cookie folgendes einfüge und das Cookie zulasse, sehe ich in der Konsole nichts:
<script>
console.log('consent GA');
</script>

Nach einem Refresh wird das console.log dann doch ausgegeben bzw. das Skript geladen. 
Ich dachte das müsste ohne Refresh gehen, no? Funktioniert das bei euch? Wenn ja, verpass ich womöglich etwas?

**Comment by bitshiftersgmbh** [comment]
Das müsste doch inzwischen gehen oder?
